cli/internal/fs/fs.go

```diff
@@ -66,7 +66,7 @@ func CopyFile(from *LstatCachedFile, to string) error {
 	fromFile, err := from.Path.Open()
 	if err != nil {
 		fromType, err := from.GetType()
```

Comment:
We're just getting Type here now, we'll need to swap that for mode:

```suggestion
		fromMode, err := from.GetMode()
```

Reply:
(Good catch!)